ubyssey/views/main.py

```diff
@@ -16,39 +16,70 @@
 from django_user_agents.utils import get_user_agent
 
 from dispatch.models import Article, Section, Subsection, Topic, Person, Podcast, PodcastEpisode, Video, Author, Image
+from django.views.generic.base import TemplateView
 
 import ubyssey
 import ubyssey.cron
 from ubyssey.helpers import ArticleHelper, PageHelper, SubsectionHelper, PodcastHelper, NationalsHelper, FoodInsecurityHelper, VideoHelper
+from ubyssey.mixins import ArticleMixin
 
 def parse_int_or_none(maybe_int):
     try:
         return int(maybe_int)
     except (TypeError, ValueError):
         return None
 
+class UbysseyHomePageView(ArticleMixin, TemplateView):
```

Comment:
👏 

Nice, I think it makes a lot of sense to use a class-based view for the homepage.